spring-cloud-dataflow-docs/src/main/asciidoc/task-developer-guide.adoc

```diff
@@ -288,35 +293,53 @@ It consists of the following procedures:
 
 Once you have Spring Cloud Data Flow Server and Shell running, you can use the following procedure to create your first task:
 
-. Register a basic suite of tasks by importing their registrations through the Spring Cloud Data Flow Shell with the following command:
-+
+1) Register a basic suite of tasks by importing their registrations through the Spring Cloud Data Flow Shell with the following command:
+
 `app register --name timestamp --type task --uri maven://org.springframework.cloud.task.app:timestamp-task:1.3.0.RELEASE`
-+
+
 NOTE: This example shows how to register a task from a Maven repository.
-. Verify that the timestamp-task app registered by running the following command in the Spring Cloud Data Flow Shell:
-+
+2) Verify that the timestamp-task app registered by running the following command in the Spring Cloud Data Flow Shell:
+
 `app list`
-+
+
 The following output should appear:
-+
-image:images/dataflow-shell-app-list.png[]
-. Create a task definition that uses timestamp task by using the following command in the Spring Cloud Data Flow Shell:
-+
+
+[source,bash,subs=attributes,options="nowrap"]
+----
+╔═══╤══════╤═════════╤════╤═════════╗
+║app│source│processor│sink│  task   ║
+╠═══╪══════╪═════════╪════╪═════════╣
+║   │      │         │    │timestamp║
+╚═══╧══════╧═════════╧════╧═════════╝
+----
+
+3) Create a task definition that uses timestamp task by using the following command in the Spring Cloud Data Flow Shell:
+
 `task create --name myStamp --definition "timestamp"`
-+
+
 You should see a message saying "Created new task 'myStamp'".
-. Launch your new task by using the following command:
-+
+
+4) Launch your new task by using the following command:
+
 `task launch myStamp`
-+
+
 You should see a message saying "Launched task `myStamp`".
```

Comment:
This might conflict with changes inflight via spring-cloud/spring-cloud-dataflow#2957